contents/handbook/strategy/roadmap.md

```diff
@@ -4,66 +4,48 @@ sidebar: Handbook
 showTitle: true
 ---
 
-## Up-to-date roadmap
+Our mission is to make more successful products in the world. 
 
-We keep our most up-to-date roadmap on [GitHub](https://github.com/orgs/PostHog/projects/1).
+Our roadmap for 2021 will do three things:
+1. Create a solid core product that's easy to use
+2. Have the best developer platform for event-based analytics
+3. Set us up to service huge volumes
 
-## General Product Strategy
+# 1. Core product
 
-### Open Core Business Model
+PostHog is a product that people love, primarily because covers 90% of analytics use cases but bundled into one package.
+Some examples of the functionality we've built last year:
 
-We aim for every company that builds software to be using PostHog.
+- Product analytics
+- Session recording
+- Feature flags
+- Heatmaps
+- Autocapture
 
-* The open source platform is designed to be installed by a developer when they start a new project, before a Product Team is in place. There is no competition at this stage.
-* The open source platform is also well suited to adoption in an existing startup, midmarket or enterprise customer as it has no information security and no vendor risk management requirements. We are much lower friction.
-* Providing self-hosted capabilities means we have complete flexibility over how we charge customers. This means we can align with higher volume customers.
+There's plenty of work to be done within those categories to make a product that is especially useful for engineers and other product minded people. To add on to that, this year we want to build:
 
-### Becoming a Piece of Infrastructure
+- A/B testing
```

Comment:
Genuine q: is this something you have to have too much volume to use for this to appeal to a startup - how many YC companies would use it, for example?

Also, how good a fit is A/B testing in a _product_ versus in a marketing website?

I'd advocate to build this but in a bare bones way to tick the box. I'd see it as a case of hitting the basics of A/B testing since we get a lot of high volume b2c startups wanting to move to us, but they're not super sophisticated although I suspect they'd want something simple. PostHog should be used on landing pages and the product, especially given we don't have to charge on volume, which makes this more useful too.

Reply:
yeah i'm not saying we're going to go deep here. It was also on your slides, and the list was pretty short without it 🤷